static/app/utils/featureFlagOverrides.ts

```diff
@@ -109,10 +115,38 @@ export default class FeatureFlagOverrides {
     return Array.from(names);
   }
 
+  /**
+   * Return list of recently accessed feature flags.
+   */
+  public getFeatureFlags() {
+    return FEATURE_FLAGS;
+  }
+
   /**
    * Stash the original list of features & override organization.features with the effective list of features
    */
   public loadOrg(organization: Organization) {
     organization.features = this.getEnabledFeatureFlagList(organization);
+    // Track names of features that are passed into the .includes() function.
+    const handler = {
+      apply: function (target, orgFeatures, flagName) {
+        // Evaluate the result of .includes()
+        const flagResult = target.apply(orgFeatures, flagName);
+
+        // Check that the flag is not already in the buffer
+        if (!FEATURE_FLAGS.values.some(f => f.flag === flagName)) {
```

Comment:
if we are returning a buffer based on evaluation order, need to think about how we handle the same flag being evaluated. do we want to keep duplicates? or if no dups, shouldn't we delete the item from the buffer and re-insert it later?

Reply:
I would check and see how the python version is doing it today. The answer to this is something that we should 100% write down, because all sdk's should be consistent imo.

Reply:
discussed offline: we should be removing duplicates, they're only temporarily on the python SDK to do some testing. will remove dups from this